docs/index.md

```diff
@@ -79,31 +61,31 @@ API Reference
 ::::{grid} 1 2 2 3
 :gutter: 1 1 1 2
 
-:::{grid-item-card} {material-outlined}`hub;1.5em;sd-mr-1` MedRecord
+:::{grid-item-card} {material-outlined}`hub;1.5em;sd-mr-1` User Guide
 :link-type: doc
 
-The {py:class}`MedRecord() <medmodels.medrecord.medrecord.MedRecord>` class is providing a straight forward and easy to use dataclass that enables you to keep your medical data in a flexible graph format implemented in Rust to make data handling as fast and efficient as possible.
+The User Guide is your go-to resource for mastering MedModels and quickly learning the essentials for analyzing complex medical data with ease.
 
 +++
-[Learn more »](./api/medrecord.md)
+[Learn more »](./user_guide/index.md)
 :::
 
-:::{grid-item-card} {material-outlined}`troubleshoot;1.5em;sd-mr-1` Treatment Effect
+:::{grid-item-card} {material-outlined}`troubleshoot;1.5em;sd-mr-1` API Reference
 :link-type: doc
 
-Use MedModels to caluclate {py:class}`TreatmentEffect() <medmodels.treatment_effect.treatment_effect.TreatmentEffect>` with various options of patient matching and controlling for confounding
+The API reference provides detailed information on all MedModels functionalities and technical interfaces, serving as a quick lookup resource.
 
 +++
-[Learn more »](./api/treatment_effect.md)
+[Learn more »](./api/index.md)
 :::
 
-:::{grid-item-card} {material-outlined}`school;1.5em;sd-mr-1` User Guide
+:::{grid-item-card} {material-outlined}`school;1.5em;sd-mr-1` Development Guide
 :link-type: doc
 
-Checkout our user guide to get strated!
+The Development Guide provides comprehensive guidelines on how to contribute to MedModels.
```

Comment:
```suggestion
The Developer Guide provides comprehensive guidelines on how to contribute to MedModels.
```

Reply:
the headline above this paragraph also says "Development Guide". We can also completely switch to Development Guide but then it has to be renamed everywhere else